src/pages/RFOX/hooks/useCurrentApyQuery.ts

```diff
@@ -49,7 +49,7 @@ export const useCurrentApyQuery = ({ stakingAssetId }: useCurrentApyQueryProps)
       if (!stakingAsset) return
       if (!totalStakedCryptoCurrencyQuery?.data) return
 
-      const previousEpoch = epochs[epochs.length - 1]
+      const previousEpoch = epochs[0]
```

Comment:
Epochs are indeed sorted from newest to oldest!

<img width="515" alt="Screenshot 2025-03-17 at 11 21 15" src="https://github.com/user-attachments/assets/dcdd9734-d076-4032-a9fe-d151431e257d" />